content/source/docs/enterprise-beta/api/workspaces.html.md

```diff
@@ -71,30 +71,31 @@ $ curl \
          }
       },
       "links":{
-         "self":"/api/v2/organizations/skierkowski/workspaces/ws-2Qhk7LHgbMrm3grF"
+         "self":"/api/v2/organizations/my-organization/workspaces/ws-2Qhk7LHgbMrm3grF"
       }
    }
 }
 ```
 
 
 
-## Create a Workspace with VCS Configuration
+## Create a Workspace with a VCS Configuration
+
+The default `/workspaces` endpoint creates a workspace without configuring the VCS connection (`ingress-trigger`). In Beta there is no dedicated endpoint for managing the `ingress-trigger` so it must be configured at the time of workspace creation.
 
-The default `/workspaces` endpoint creates a workspace without configuring the VCS connection (`ingress-trigger`). In Beta there is no dedicated endpoint for managing the `ingress-trigger`so it must be configured at the time of workspace creation.
+| Method | Path           |
+| :----- | :------------- |
+| POST | /organizations/:organization/compound-workspaces |
 
-**Method**: POST
 
-**Path**: /organizations/:organization/compound-workspaces
 
 ### Parameters
 
-- `name` (string: \<required\>) - Specifies the name of the workspace. This must be a alphanumeric and `-`or `_`. This will be used as an identifier and must be unique in the organization.
-- `environment`(string: "default") - Specifies the environment attribute on the workspace.
-- `organization` (string: \<required\>) - Specififes the username or organizaiton name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This is specified in the URL path.
-- `default-branch`(boolean: true) - specifies if the default branch should be used. In the beta release this is set to `true` by default and it is not configurable. Providing a branch will result in an error.
-- `path`(string:'/') - Specifies the directory of the repo to be used for the workspaces. Only this directory is cloned and the root path and other directories are not cloned.
-- `linkable-repo-id`(string: \<required\>) - This is the ID of the repository to be used. The ID can be obtained from the `linkable-repos`endpoint.
+- `:organization` (`string: <required>`) - Specififes the username or organizaiton name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This is specified in the URL path.
+- `name` (`string: <required>`) - Specifies the name of the workspace. This must be a alphanumeric and `-`or `_`. This will be used as an identifier and must be unique in the organization.
+- `default-branch`(`boolean: true`) - specifies if the default branch should be used. In the beta release this is set to `true` by default and it is not configurable. Providing a branch will result in an error.
```

Comment:
hmmm, i thought you _could_ provide a branch if you wanted to and it would work fine?

Reply:
It's been fixed. I'll update the doc.